benchmarks/tpch/polars_queries/q16.py

```diff
@@ -0,0 +1,43 @@
+
+from datetime import datetime
+
+import polars_queries.utils as polars_utils
+import polars as pl
+from linetimer import CodeTimer, linetimer
+
+Q_NUM = 16
+pl.Config().set_fmt_float("full")
+
+
+@linetimer(name=f"Overall execution of polars Query {Q_NUM}", unit="ms")
+def q():
```

Comment:
why not call these functions `main`? 

^ wrote the above before I realized that this is all imported queries. 

do we need to commit these or can we checkthem out as part of the setup from the polars repo?

Reply:
> do we need to commit these or can we checkthem out as part of the setup from the polars repo?

not sure I follow? 

are you suggesting importing them from the https://github.com/pola-rs/tpch repo?